### PR TITLE
add webpack config + webpack-dev-server

### PR DESCRIPTION
**Pull request by KrivichaninDenis**
**Project start** - `npm start`
**[webpack.config.js](https://github.com/pub-t/plain-destination/blob/webpack/webpack.config.js)** - webpack config

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by КРИВИЧАНИН on 24.06.2016.
```
Comment (irnc) [inline webpack.config.js]: Could we please remove such commits, they gets outdated very soon, and may discourage new committers.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ */
+module.exports = {
+  entry: './js/script.js',
+  output: {
+    path: __dirname,
```
Comment (irnc) [inline webpack.config.js]: Putting auto-generated code into the same directory as human code is discouraged. Please put build artifacts into a separate directory and add it's content to `.gitignore`

**Comment by KrivichaninDenis** [comment]
Create bundle [directory](https://github.com/pub-t/plain-destination/pull/32/commits/19a8b8b776dc6e1cc9fd05b5e1a8754fd9cc25e8#diff-11e9f7f953edc64ba14b0cc350ae7b9dR4) and update `.gitignore`

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  entry: './js/script.js',
+  output: {
+    path: './bundleDirectory',
```
Comment (irnc) [inline webpack.config.js]: `/dist` is a common name used for such purposes, derived form _distribution_. Please rename, as it makes no sense to name directory a directory.

**Comment by KrivichaninDenis** [comment]
All understand why we need [webpack](https://github.com/pub-t/plain-destination/blob/webpack/webpack.config.js)?

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,7 @@
 .idea
 
 #ignore node_modules
-node_modules
\ No newline at end of file
+node_modules
+
+#ignore webpack bundle files
+bundleDirectory
```
Comment (irnc) [inline .gitignore]: This should be `dist`, and while you are on that line, please add newline at the end of file.

**Pull request merged**